lint-staged.config.js

```diff
@@ -0,0 +1,7 @@
+const formatFiles = "prettier --ignore-unknown --write";
+const sortPackageJson = "better-sort-package-json";
+
+export default {
+  "socials.json|*.{js,md,ts,yml}": [formatFiles],
```

Comment:
I think that would work better:

```suggestion
  "!(package.json)": [formatFiles],
```

Reply:
Done here [b75e757](https://github.com/hemilabs/hemi-socials/pull/1/commits/b75e7578cc8cf067ae4034f8b8e74eed54d148ad)